### PR TITLE
Clean repository & prettier

### PR DESCRIPTION
**Pull request by Max-7**

### CODE REVIEW

File: lib/components/ms-modal/MsTextInputModal.vue [hunk 1]
```diff
@@ -64,4 +64,4 @@ function cancel(): Promise<boolean> {
 </script>
 
 <style scoped lang="scss"></style>
-@lib/common/asyncComputed@lib/common/validators
\ No newline at end of file
+@lib/common/asyncComputed@lib/common/validators
```
Comment (NicoTuxx) [inline lib/components/ms-modal/MsTextInputModal.vue:67]: should be removed

File: package.json [hunk 2]
```diff
@@ -17,50 +17,25 @@
...
     "build:lib": "vite build",
     "lint": "eslint . --max-warnings=0 && vue-tsc --noEmit",
     "prepublishOnly": "npm run build",
     "prepare": "npm run build",
     "test:unit": "vitest run --dom",
-    "cy:e2e:headless": "cypress run --e2e",
-    "cy:e2e": "cypress run --e2e --headed",
-    "test:e2e": "start-server-and-test dev http://localhost:8080 cy:e2e",
-    "test:e2e:headless": "start-server-and-test dev http://localhost:8080 cy:e2e:headless",
-    "web:open": "vite --open",
```
Comment (NicoTuxx) [inline package.json:31]: should not be remove, is used to launch local app to test components

File: package.json [hunk 3]
```diff
@@ -17,50 +17,25 @@
...
   },
   "dependencies": {
-    "@capacitor/app": "5.0.7",
-    "@capacitor/core": "5.7.4",
-    "@capacitor/haptics": "5.0.7",
-    "@capacitor/keyboard": "5.0.8",
-    "@capacitor/status-bar": "5.0.7",
     "@ionic/vue": "^7.0.0",
-    "@ionic/vue-router": "^7.0.0",
-    "@types/luxon": "^3.3.4",
```
Comment (NicoTuxx) [inline package.json:55]: is it not useful since we use luxon?

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Max-7) [reply package.json:31]: Ah, yeah. I kept the `dev` but was a bit quick with the `web:open`.

File: package.json (hunk 3 above)
Comment (Max-7) [reply package.json:55]: It is but I moved it to the `devDependencies`. I don't see why it would be needed apart from dev.

**Review by NicoTuxx — APPROVED** [review]

**Pull request merged**